fabtests/include/unix/osd.h

```diff
@@ -94,6 +94,11 @@ OFI_COMPLEX_TYPE_DECL(float, float)
 OFI_COMPLEX_TYPE_DECL(double, double)
 OFI_COMPLEX_TYPE_DECL(long_double, long double)
 
+/* CMPLX may be undefined before C11 */
+#ifndef CMPLX
+#define CMPLX(r, i) ((double complex)((double)(r) + _Imaginary_I * (double)(i)))
```

Comment:
Oops, sorry!
What is the method used to define an atomic before C11? Does this work for other types (like long double)?

Reply:
The macro here is what is defined in the C11 header.

Reply:
So this should be equivalent to the original code. There is CMPLXF and CMPLXL for float and long complex numbers.

Reply:
```
#define CMPLX(x, y) ((double complex)((double)(x) + _Imaginary_I * (double)(y)))
#define CMPLXF(x, y) ((float complex)((float)(x) + _Imaginary_I * (float)(y)))
#define CMPLXL(x, y) ((long double complex)((long double)(x) + \
                      _Imaginary_I * (long double)(y)))
```

Reply:
`_Imaginary_I` is defined since C99.

Reply:
Updated. Should handle other types better now.